.github/workflows/ci.yml

```
@@ -55,14 +55,8 @@ jobs:
     if: github.event_name == 'push' && contains(github.ref, 'refs/tags/')
     steps:
       - uses: actions/checkout@v2
-      - name: Set up Python 3.9
-        uses: actions/setup-python@v2
-        with:
-          python-version: 3.9
-      - name: Install build
-        run: python -m pip install --user build
       - name: Build sdist and wheel
-        run: python -m build
+        run: pipx run build
```

Shouldn't this be `pyproject-build` or am I missing something?

```suggestion
        run: pipx run pyproject-build
```

---

Oh I see you got this under control :) https://github.com/pypa/pipx/issues/600